browser/base/content/utilityOverlay.js

```diff
@@ -640,7 +640,7 @@ function openLinkIn(url, where, params) {
         targetBrowser.currentURI.spec != "about:blank"
       ) {
         w.gBrowser.updateBrowserRemoteness(targetBrowser, {
-          recordExecution: "*",
+          recordExecution: browser.hasAttribute("recordExecution"),
```

Comment:
Should this be getAttribute?

Reply:
Yep! I noticed right after hitting submit but you beat me to it :P